packages/binding-coap/test/coap-server-test.ts

```diff
@@ -202,6 +202,35 @@ class CoapServerTest {
         await coapServer1.stop();
     }
 
+    @test async "should support IPv6"() {
+        const coapServer = new CoapServer(PORT, "::");
+        await coapServer.start(null);
+
+        const testThing = new ExposedThing(null, {
+            title: "Test",
+            properties: {
+                test: {
+                    type: "string",
+                    forms: [],
+                },
+            },
+        });
+
+        const test: DataSchemaValue = "testValue";
+        testThing.setPropertyReadHandler("test", (_) => Promise.resolve(test));
+
+        await coapServer.expose(testThing);
+
+        const uri = `coap://[::1]:${coapServer.getPort()}/test/`;
+
+        const coapClient = new CoapClient();
```

Comment:
I recall that the issue was especially caused by the usage of the server as an argument in the coap client. See:
```ts
// create Servient and add CoAP binding
const servient = new Servient();
servient.addClientFactory(new CoapClientFactory(new CoapServer())); // you have to import CoapServer
```
Can you update the test to closely resemble what would happen if we are using the `CoapClientFactory` ?

Reply:
Oh, sure, in an earlier version I actually reused the socket already, I will update the PR shortly :)

Reply:
yes, it does!

Reply:
Great! :) Thank you for approving, I took the freedom of squashing the commits into one :)